icons/platformio.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>PlatformIO icon</title><path d="m16.43591,5.33155l0.69738,-2.51724c0.71412,-0.06342 1.27426,-0.66304 1.27426,-1.39527c0,-0.77374 -0.62709,-1.40103 -1.40034,-1.40103c-0.77326,0 -1.39923,0.62729 -1.39923,1.40103c0,0.50968 0.27226,0.95478 0.67953,1.19924l-0.68511,2.47227c-1.95825,-0.49814 -3.61188,-0.46125 -3.61188,-0.46125l-0.30573,0.13837l0,19.13245l0.30573,0.10263c1.84667,0 9.92627,-5.58683 9.94523,-11.58186c0,-4.29419 -2.8509,-6.22335 -5.50095,-7.08934l0.00112,0zm-1.94486,12.32447c-1.57553,-2.5207 -0.90939,-8.38659 3.20684,-9.2249c1.12809,-0.18219 2.51058,0.56041 2.91785,2.21052c1.02766,3.49854 -5.02115,8.39927 -6.12581,7.01438l0.00112,0zm2.47041,-5.39887c-0.70296,0 -1.27426,0.57079 -1.27426,1.2765c0,0.70455 0.57018,1.2765 1.27426,1.2765c0.70408,0 1.27537,-0.57194 1.27537,-1.2765c0,-0.7057 -0.5713,-1.2765 -1.27537,-1.2765zm0.38161,1.17733a0.31689,0.32748 0 0 1 -0.32247,-0.32287c0,-0.17758 0.14506,-0.32287 0.32247,-0.32287c0.17853,0 0.32135,0.14529 0.32135,0.32287c0,0.17873 -0.14394,0.32287 -0.32135,0.32287zm-8.97782,-8.28627l-0.68065,-2.56683c0.38942,-0.24907 0.64829,-0.68495 0.64829,-1.18194c0,-0.77374 -0.62597,-1.40218 -1.39923,-1.40218s-1.39923,0.62845 -1.39923,1.40218c0,0.77259 0.62262,1.39757 1.39253,1.40103l0.68288,2.5749c-2.6601,0.88328 -5.54559,2.81705 -5.54559,7.04552c0.05914,6.11265 8.0796,11.58071 9.92627,11.58071l0,-19.37229s-1.66144,-0.00807 -3.62527,0.5189zm1.41039,12.50781c-1.10465,1.38374 -7.15236,-3.51584 -6.12469,-7.01438c0.40727,-1.64895 1.78865,-2.39271 2.91785,-2.21167c4.11623,0.83947 4.78237,6.70535 3.20684,9.2249l0,0.00115zm-2.55298,-5.35967c-0.70296,0 -1.27426,0.57079 -1.27426,1.2765c0,0.70455 0.57018,1.2765 1.27426,1.2765c0.70408,0 1.27537,-0.57194 1.27537,-1.2765c0,-0.7057 -0.5713,-1.2765 -1.27537,-1.2765zm-0.38049,1.17733a0.31689,0.32748 0 0 1 -0.32359,-0.32287c0,-0.17758 0.14506,-0.32172 0.32359,-0.32172c0.17741,0 0.32135,0.14414 0.32135,0.32287c0,0.17758 -0.14506,0.32287 -0.32135,0.32287l0,-0.00115z"/></svg>
```

Comment:
Should be <title>PlatformIO</title>, not <title>Platform IO icon</title>. Also looks like the linter is failing because the icon size is off, which usually indicates that you need to optimize at a higher precision (perhaps 4).